From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bjarne Koll <lynxplay101@gmail.com>
Date: Thu, 3 Mar 2022 00:20:36 +0100
Subject: [PATCH] Build changes

Initial build changes done to the build file, modifying dependencies and
the likes.

diff --git a/build.gradle.kts b/build.gradle.kts
index 92b612126a6bac0b89198a92bbb73b742ec9d064..72b8557d25b353936eb0f43fbf0bde388629b364 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -30,7 +30,7 @@ dependencies {
     api("com.google.code.gson:gson:2.8.8") // Paper - bump
     api("net.md-5:bungeecord-chat:1.16-R0.4")
     api("org.yaml:snakeyaml:1.28")
-    api("com.googlecode.json-simple:json-simple:1.1.1") // Paper
+    api("com.googlecode.json-simple:json-simple:1.1.1") { exclude("junit", "junit") } // Paper // KTP - drop junit dependency
     api("it.unimi.dsi:fastutil:8.2.2")
     apiAndDocs(platform("net.kyori:adventure-bom:$adventureVersion"))
     apiAndDocs("net.kyori:adventure-api")
